src/components/App/app.js

```diff
@@ -15,6 +15,7 @@ import { setThemeMode } from '../../redux/theme/theme.actions';
 import { getCategories } from '../../redux/categories/categories.actions';
```

Comment:
Please, revert this renaming and move this component to the `containers` folder because it is connected to the redux

Reply:
Done